configservice/config-service.smithy

```diff
@@ -65,5 +65,9 @@ structure RegistryCredential {
     token: String,
     /// If supplied, username and password will be used for HTTP Basic authentication
     username: String,
-    password: String
+    @sensitive
```

Comment:
Just a heads-up that the last time I looked at the generated code, this annotation had no effect.

Reply:
It doesn't currently have an effect, but Steve requested I add it so if in the future we do generate different code based off this, the smithy definition won't need to be updated